website/docs/advanced/BrownfieldIntegration.md

```diff
@@ -212,18 +224,29 @@ void saveStorageValue(Context ctx, String key, String value) {
   AsyncStorageAccess asyncStorage = StorageModule.getStorageInstance(ctx);
 
   BuildersKt.launch(GlobalScope.INSTANCE,
-          Dispatchers.getIO(),
-          CoroutineStart.DEFAULT,
-          (scope, continuation) -> {
-
-            List<Entry> entries = new ArrayList<>();
-            Entry entry = new Entry(key, value);
-            entries.add(entry);
-
-            asyncStorage.setValues(entries, continuation);
-
-            return Unit.INSTANCE;
-          });
+                Dispatchers.getIO(),
+                CoroutineStart.DEFAULT,
+                (scope, continuation) -> {
+                    List<String> keys = new ArrayList<>();
+                    keys.add("kkk");
+
+                    Continuation cont = new Continuation() {
+                        @NotNull
+                        @Override
+                        public CoroutineContext getContext() {
+                            return scope.getCoroutineContext();
+                        }
+
+                        @Override
+                        public void resumeWith(@NotNull Object o) {}
+                    };
+
+                    List<Entry> entries = new ArrayList<>();
+                    Entry entry = new Entry("my_key", "my_value");
```

Comment:
In this psuedo-method example, we have `key` and `value` arguments, so let's us it here:
```suggestion
                    Entry entry = new Entry(key, value);
```